src/Features/Core/Portable/AddImport/SymbolReferenceFinder_PackageAssemblySearch.cs

```diff
@@ -154,6 +142,10 @@ private partial class SymbolReferenceFinder
                 {
                     cancellationToken.ThrowIfCancellationRequested();
                     var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
+
+                    // The project must support compilations since we only create compilation for the project that invoked AddImport feature.
+                    RoslynDebug.Assert(compilation != null);
```

Comment:
can we call RequireCompilationAsync intead?

Reply:
You mean `GetRequiredCompilationAsync`? Yes. I didn't notice we have that. Why do we declare it as an extension method 🤦‍♂?